### DISCUSSION TITLE
Request for reports with new Fujifilm firmware

### DISCUSSION THREAD

**Discussion by gkoh** [gkoh/furble#47] [Q&A]
Hello all,

After looking into #46, I need to crowd source data for whether it is an isolated incident or specific to some cameras/firmwares.
I have an X-T30, which works as before with firmware v2.01.

I am particularly interested in reports from other X-T4 users that have upgraded to firmware v2.00.
Happy to field reports from other camera models that are also working/not with the latest v2 firmware.

**Comment by pstawski** [comment]
Tested X-S20 with 1.13 firmware without issues so far. Will upgrade to the latest version and report.

**Comment by gkoh** [comment]
I have not received any further reports of issues, thus closing this discussion.
